qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -200,6 +200,31 @@
             gagent_check_type = fsfreeze
             gagent_fs_test_cmd = "echo foo > %s/foo"
             mountpoint_def = "/tmp"
+            virt_test_type = qemu
+            images += " data"
+            force_create_image_data = yes
+            force_remove_image_data = yes
```

Comment:
remove_image_data = yes
Refer to my search results, force_remove_image_data is used for lvm in VT

Reply:
It's not related to lvm, `_data` suffix will be dropped after `params.object_params` since `images += " data"`.

Reply:
Thanks @PaulYuuu for pointing this out, I mean force_remove_image param is used for lvm module in vt, and remove_image param is used for image remove in postprocess_image, to point this out, because I found some cases mis-used these two, although image remove is a default way in vt

Reply:
Oh, sorry, I confused a typo.

Reply:
Never mind, and your inputs made the comment more clearly.

Reply:
will delete force_remove_image_data = yes

Reply:
fixed